kolibri/plugins/device/assets/src/modules/deviceInfo/handlers.js

```diff
@@ -2,7 +2,7 @@ import client from 'kolibri.client';
 import urls from 'kolibri.urls';
 import samePageCheckGenerator from 'kolibri.utils.samePageCheckGenerator';
 import bytesForHumans from 'kolibri.utils.bytesForHumans';
-import { isEmbeddedWebView } from 'kolibri.utils.browserInfo';
+import store from 'kolibri.coreVue.vuex.store';
```

Comment:
Would be preferable to still use the composable here.

Can import `import useUser from 'kolibri.coreVue.composables.useUser';`

To be able to use the `isAppContext` ref outside of a Vue component (or within the setup method of a Vue component) we need to be explicit that we want the value of it. There is a special function `get` importable like this that we can use:

```
import { get } from '@vueuse/core';
```

and then in the module scope do:
```
const { isAppContext } = useUser();
```

I have marked below how to use `get` and `isAppContext` to check the value.